app/services/imports/sales_logs_import_service.rb

```diff
@@ -242,7 +243,8 @@ def fields_not_present_in_softwire_data
          staircase_bought_value_check
          monthly_charges_value_check
          hodate_check
-         saledate_check]
+         saledate_check
+         student_not_child_value_check]
```

Comment:
nit: can we put the ] on the next line for tidiness

Reply:
Oh, rubocop don't like that :<